pykickstart/sections.py

```diff
@@ -683,6 +684,21 @@ def _getParser(self):
                         Recommends and Supplements flags. By default weak
                         dependencies will be installed.""")
 
+        if self.version < F32:
+            return op
+
+        op.add_argument("--ignorebroken", action="store_true", default=False, version=F32,
+                        help="""
+                        Ignore any packages, groups or modules with conflicting files.
+                        This issue will practically disable DNF strict option.
+                        The default behavior will abort the installation with error
+                        message describing the conflicting files.
+
+                        **WARNING: Usage of this parameter is DISCOURAGED! The DNF
+                        strict option will NOT log any information about what packages
+                        were skipped. The resulting system may be in the undefined behavior!**
```

Comment:
Replace last sentence with "Using this option may result in an unusable system."